recipe/conda_build_config.yaml

```diff
@@ -1,4 +1,8 @@
-MACOSX_DEPLOYMENT_TARGET:  # [osx and x86_64]
-  - '10.13'                # [osx and x86_64]
-MACOSX_SDK_VERSION:        # [osx and x86_64]
-  - '10.14'                # [osx and x86_64]
+# https://doc.qt.io/qt-6/macos.html
+c_stdlib_version:          # [osx]
+  - 11.0                   # [osx]
+MACOSX_SDK_VERSION:        # [osx]
+  - '12.3'                 # [osx]
+  
+c_stdlib_version:          # [linux]
+  - 2.17                   # [linux]
```

Comment:
```suggestion
```

I don't think we need this any more (it got dropped by @jakirkham while getting Matplotlib to build last week).

Reply:
The SDK portion most likely needs to be kept (if it was necessary before), and osx-64 does not yet have 11.0 as the lower bound, so it needs to be kept.
```suggestion
# https://doc.qt.io/qt-6/macos.html
c_stdlib_version:          # [osx and x86_64]
  - 11.0                   # [osx and x86_64]
MACOSX_SDK_VERSION:        # [osx]
  - '12.3'                 # [osx]
```